kratos/python_scripts/assign_flag_process.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+# Importing the Kratos Library
+import KratosMultiphysics
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return AssignFlagProcess(Model, settings["Parameters"])
+
+# All the processes python processes should be derived from "Process"
+
+class AssignFlagProcess(KratosMultiphysics.Process):
+    """This process sets a given value for a certain flag in all the nodes of a submodelpart
+
+    Only the member variables listed below should be accessed directly.
+
+    Public member variables:
+    Model -- the container of the different model parts.
+    settings -- Kratos parameters containing solver settings.
+    """
+    
+    def __init__(self, Model, settings ):
+        """ The default constructor of the class
+        
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing solver settings.
+        """
+        
+        KratosMultiphysics.Process.__init__(self)
+
+        #The value can be a double or a string (function)
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "help"            : "This process sets a given value for a certain flag in all the given entities of a submodelpart",
+            "mesh_id"         : 0,
+            "model_part_name" : "please_specify_model_part_name",
+            "flag_name"       : "SPECIFY_FLAG_NAME",
+            "interval"        : [0.0, 1e30],
+            "value"           : true,
+            "entities"        : ["nodes","conditions","elements"]
```

Comment:
are you filling this already just for documentation?
I think it is a bad idea bcs if the user does not specify `entities` then these will be the defaults that are being used!

Reply:
I know, that why by default is everything

Reply:
I don't think it is a good idea to give defaults here, this can silently change stuff one does not intend and is difficult to debug
=> can you make this empty and throw an error later when you check if it is empty?
=> this way the user has to know which entities the flag is assigned to

Reply:
OK, I can